Resources/ServerInfo/Guidebook/Antagonist/Nuclear Operatives.xml

```diff
@@ -0,0 +1,53 @@
+<GuideEntityEmbed Entity="ClothingOuterHardsuitSyndieElite" Caption="Agent Hardsuit"/>
+<GuideEntityEmbed Entity="ClothingOuterHardsuitSyndie" Caption="Regular Hardsuit"/>
+</Box>
+
+## Preparation
+
+Taking on a whole station in a fight is a difficult job, luckily you are well prepared for the job. Every operative has an [color=#a4885c]Uplink[/color]. It comes with 40 [color=#a4885c]Telecrystals[/color], which you can use to purchase gear. What you get depends on the strategy picked by your commander or voted for by the team. Only your commander and agent will have passenger access, but no external airlock access. This means you will need something to hack or blast your way onto the station.  In general you should need some weapons, something to breach doors, and utility/healing.
+
+<Box>
+<GuideEntityEmbed Entity="BaseUplinkRadio40TC" Caption="The Uplink"/>
+</Box>
+
+## Getting to the Station
+
+After gearing up and creating a plan, grab a jetpack from your armory and go with your other operatives to the [color=#a4885c]Nukie Shuttle[/color]. Here you will find some extra explosives and tools, as well as your nuke and nuke codes.
+Go to the head of the shuttle and interact with the IFF console. Turn [color=#a4885c]Show IFF[/color] and [color=#a4885c]Show Vessel[/color] off, so that nobody using a mass scanner on the station can see your ship. When everyone is ready, FTL to the station and fly to it with a jetpack. Don't forget the nuclear fission explosive on your ship if you are going to use it, and definitely don't forget the nuke codes or pinpointer.
+
+## The Disk
+
+You will notice that each operative starts with a [color=#a4885c]Pinpointer[/color]. This device is one of the most important to the mission, and you should keep it with you at all times. Turn on the pinpointer when you arrive to the station and it will always point to the [color=#a4885c]Nuke Disk[/color], your next objective.
+The nuke disk belongs to the captain, they will almost always have it on them. Follow the direction of the arrow on the pinpointer to the captain, and do whatever it takes to get it from them.
+
+<Box>
+<GuideEntityEmbed Entity="PinpointerNuclear" Caption="The Pinpointer"/>
+<GuideEntityEmbed Entity="NukeDisk" Caption="The Nuke Disk"/>
+</Box>
+
+## The Nuke
+
+On your shuttle you will find one of two [color=#a4885c]Nuclear Fission Explosives[/color] and a paper with the nuke codes. The paper will tell you which explosive it corresponds to. If the ID is the same as the one on your nuke, the codes will only work for it. If it is not, then they will only work for the explosive in the station's vault.
+
+When you acquire the nuke disk, put it into the explosives and use the code to arm the nuke. It takes 30 seconds for a crewmember to disarm, and will count down from 300. Be prepared to defend the nuke for as long as possible, remember that escaping alive isn't necessary, but recommended.
```

Comment:
```suggestion
Once you acquire the nuke disk, put it inside the nuke and use the code to arm it. It takes 30 seconds for a crewmember to disarm it, and it will count down from 300. Be prepared to defend the nuke for as long as possible, remember that escaping alive isn't necessary, but recommended.
```